source/shared/sheets.js

```diff
@@ -178,3 +197,73 @@ export function showTouchIDToggleSheet() {
         );
     });
 }
+
+export function showAutoFillToggleSheet() {
+    const state = getState();
+    const currentSourceID = getSelectedSourceID(state);
+    const itemEnableAutoFill = "Enable Autofill";
+    const itemDisableAutoFill = "Disable Autofill";
+    const itemSystemSettings = "Android Settings";
+    const itemCancel = "Cancel";
+    return Promise.all([getAutoFillSystemStatus(), autoFillEnabledForSource(currentSourceID)]).then(
+        results => {
```

Comment:
maybe destructure this instead of assigning new constants later:

```suggestion
        ([autoFillEnabled, archiveEnabled]) => {
```